src/network_generator.py

```
@@ -157,7 +156,7 @@ def calculate_travel_times(
             return True
 
         except Exception as e:
-            logger.error(f"Error occurred while calculating travel times: {e}", exc_info=True)
+            logger.exception(f"Error occurred while calculating travel times: {e}")
```

Ruff (0.14.2)

Redundant exception object included in `logging.exception` call

(TRY401)